create-planar-and-geodetic-buffers/src/main/res/layout/activity_main.xml

```diff
@@ -27,36 +27,36 @@
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent">
 
-            <EditText
-                android:id="@+id/bufferInput"
+            <com.google.android.material.button.MaterialButton
+                android:id="@+id/optionsButton"
                 android:layout_width="0dp"
                 android:layout_height="wrap_content"
-                android:layout_marginStart="32dp"
-                android:layout_marginEnd="16dp"
-                android:hint="@string/Enter_miles_string"
-                android:inputType="number"
-                android:text="@string/default_buffer_distance"
-                app:layout_constraintEnd_toStartOf="@id/unitsDisplay"
-                app:layout_constraintStart_toStartOf="parent"
+                android:text="Buffer options"
+                app:layout_constraintBottom_toTopOf="@+id/bufferLegend"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintHorizontal_bias="0.5"
+                app:layout_constraintStart_toEndOf="@+id/clearButton"
                 app:layout_constraintTop_toTopOf="parent" />
 
-            <TextView
-                android:id="@+id/unitsDisplay"
-                android:layout_width="wrap_content"
+            <com.google.android.material.button.MaterialButton
+                android:id="@+id/clearButton"
+                style="?attr/materialButtonOutlinedStyle"
+                android:layout_width="0dp"
```

Comment:
We can have the clearButton disabled by default and enable it when the map is tapped.
`android:enabled="false"`